Gemfile.lock

```diff
@@ -121,6 +121,8 @@ GEM
     coderay (1.1.3)
     concurrent-ruby (1.1.10)
     connection_pool (2.2.5)
+    coverband (5.2.4)
+      redis (>= 3.0)
```

Comment:
Do we have the right configuration for this in prod? i.e. `ENV['REDIS_URL']`?

Reply:
Not yet, but when this gets merged I'll spin up a REDIS server

Reply:
> We'd also need to make sure we have Redis as part of our setup instructions.

Do you mean locally? If so, I'am wondering if we should keep this a production dependency and activate it only there. It doesn't make a whole lot of sense to include this in non-prod environments IMO.

Reply:
If there's a way to do that, we probably should.